types/packages/caver-rpc/src/governance.d.ts

```diff
@@ -76,6 +76,7 @@ export class Governance {
     getMyVotingPower(callback?: (error: Error, result: number) => void): Promise<number>
     getMyVotes(callback?: (error: Error, result: MyVote[]) => void): Promise<MyVote[]>
     getChainConfig(callback?: (error: Error, result: ChainConfig) => void): Promise<ChainConfig>
+    getChainConfig(blockNumber: BlockNumber, callback?: (error: Error, result: ChainConfig) => void): Promise<ChainConfig>
```

Comment:
these two lines (78, 79) implies that we can invoke getChainConfig with/without arguments, right?

Reply:
Thank you for your kind and detailed review.